lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt

```diff
@@ -240,7 +240,7 @@ abstract class MangaThemesia(
 
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
         document.selectFirst(seriesDetailsSelector)?.let { seriesDetails ->
```

Comment:
```suggestion
        document.selectFirst(seriesDetailsSelector)!!.let { seriesDetails ->
```

Reply:
still throwing null pointer with only that

Reply:
this should be there regardless